From 145a32312ca5598ea7fbc62481263323e4fedeeb Mon Sep 17 00:00:00 2001
From: Michael Rudolf <michael.rudolf01@sap.com>
Date: Sun, 13 Dec 2015 23:37:16 +0100
Subject: [PATCH] Issue 112: respect notoc option also for lists of figures and
 tables.

---
 tufte-common.def | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/tufte-common.def b/tufte-common.def
index e3f2005..fd64382 100644
--- a/tufte-common.def
+++ b/tufte-common.def
@@ -1707,6 +1707,8 @@
     {\hspace*{6em}} % before w/o label
     {\rmfamily\upshape\qquad\thecontentspage} % filler + page (leaders and page num)
     [\end{fullwidth}] % after
+  \renewcommand*\l@figure{\@tufte@lof@line}
+  \let\l@table\l@figure
 }{}
 
 %%
@@ -1747,9 +1749,6 @@
   \par%
 }
 
-\renewcommand*\l@figure{\@tufte@lof@line}
-\let\l@table\l@figure
-
 
 %%
 % A handy command to disable hyphenation for short bits of text.
