From eff58a4b07bc0a43061aa7effe0ef62c1f62f627 Mon Sep 17 00:00:00 2001
From: Michael Rudolf <michael.rudolf01@sap.com>
Date: Tue, 15 Dec 2015 23:22:11 +0100
Subject: [PATCH] Issue 87: compatibility with listings, subfigure and
 subcaption packages.

---
 tufte-common.def | 25 +++++--------------------
 1 file changed, 5 insertions(+), 20 deletions(-)

diff --git a/tufte-common.def b/tufte-common.def
index e3f2005..1212988 100644
--- a/tufte-common.def
+++ b/tufte-common.def
@@ -1241,16 +1241,13 @@
     \ifthenelse{\equal{#1}{b}\OR\equal{#1}{B}}%
       {\renewcommand{\floatalignment}{b}\@tufte@float@debug{Presumed position: [bottom]}}%
       {\renewcommand{\floatalignment}{t}\@tufte@float@debug{Presumed position: [top]}}%
-    % Capture the contents of the \caption and \label commands to use later
+    % Capture the contents of the \caption command to use later
     \global\let\@tufte@orig@caption\caption%
     \global\let\@tufte@orig@label\label%
-    \renewcommand{\caption}{\optparams{\@tufte@caption}{[][0pt]}}%
-    \renewcommand{\label}[1]{\@tufte@label{##1}}%
-    % Handle subfigure package compatibility
-    \ifthenelse{\boolean{@tufte@packages@subfigure}}{%
-      % don't move the label while inside a \subfigure or \subtable command
-      \global\let\label\@tufte@orig@label%
-    }{}% subfigure package is not loaded
+    \renewcommand{\caption}{%
+      % Now also capture the contents of the \label command to use later
+      \let\label\@tufte@label%
+      \optparams{\@tufte@caption}{[][0pt]}}%
     \@tufte@orig@float{#2}[#1]%
     \ifthenelse{\boolean{@tufte@float@star}}%
       {\setlength{\@tufte@float@contents@width}{\@tufte@fullwidth}}%
@@ -1911,18 +1908,6 @@
 }
 
 
-%%
-% Detect if the subfigure package has been loaded
-
-\newboolean{@tufte@packages@subfigure}
-\setboolean{@tufte@packages@subfigure}{false}
-\AtBeginDocument{%
-  \@ifpackageloaded{subfigure}
-    {\gsetboolean{@tufte@packages@subfigure}{true}}
-    {\gsetboolean{@tufte@packages@subfigure}{false}}%
-}
-
-
 %%
 % Detect of the float package has been loaded
 
